src/compositions/usePagination.ts

```diff
@@ -74,7 +74,7 @@ export function usePagination<
   const page = getPageRef()
   const pages = computed(() => Math.ceil(total.value / getLimit()))
 
-  const countSubscriptionParameters = computed(() => {
+  const countSubscriptionParameters = toRef(() => {
```

Comment:
using `toRef` rather than a computed ensures this callback gets re-evaluated when the value is accessed. This is important for how we do relative dates to work.